tests/test_backend.py

```diff
@@ -4,7 +4,9 @@
 import os
 import platform
 import re
+import shutil
 import subprocess
+from pathlib import Path
```

Comment:
nit: Let's keep only import modules

Reply:
I'll add fixes commit after commit and let you squash it afterwards.